geom/dcel_label.go

```diff
@@ -61,6 +61,7 @@ func newLocationsOnBoundary(operand operand) [2]location {
 	return locs
 }
 
+// nolint
```

Comment:
quite a few constants in this block are not used.

Reply:
I think we can just remove the unused constants here, but happy to do that as a second pass in order to get the base golangci lint change out first.